ReactiveCocoaTests/KeyValueObservingSpec.swift

```diff
@@ -154,6 +154,19 @@ class KeyValueObservingSpec: QuickSpec {
 				expect(weakOriginalInner).to(beNil())
 			}
 
+			describe("special tests") {
```

Comment:
This `describe` is probably not needed—I think the `it` could be at the same level as the others.